test/functional/combine_logs.py

```diff
@@ -78,16 +83,21 @@ def read_logs(tmp_dir):
     for each of the input log files."""
 
     # Find out what the folder is called that holds the debug.log file
-    chain = glob.glob("{}/node0/*/debug.log".format(tmp_dir))
+
+    chain = glob.glob(os.path.join(tmp_dir, "node0", "*", "debug.log"))
     if chain:
-        chain = chain[0]  # pick the first one if more than one chain was found (should never happen)
-        chain = re.search(r'node0/(.+?)/debug\.log$', chain).group(1)  # extract the chain name
+        # pick the first one if more than one chain was found (should never happen)
+        chain = chain[0]
+        sep = os.path.sep
+        chain = re.search(r'node0' + sep + '(.+?)' + sep +
```

Comment:
Would prefer we just use [pathlib](https://docs.python.org/3.5/library/pathlib.html) here.